include/osmium/io/writer.hpp

```diff
@@ -236,9 +243,12 @@ namespace osmium {
                     (set_option(options, args), 0)...
                 };
 
-                if (options.header.get("generator") == "") {
-                    options.header.set("generator", "libosmium/" LIBOSMIUM_VERSION_STRING);
-                }
```

Comment:
I was preparing the patch using 2.12.2, then diff-ported to master, loosing some post-2.12.2 changes, apparently. I'll restore it.